src/trustlist.rs

```diff
@@ -9,12 +8,13 @@ use thiserror::Error;
 /// Keys are indexed by their `kid` (Key ID) which is an arbitrary sequence of bytes.
 #[derive(Debug)]
 pub struct TrustList {
-    keys: HashMap<Vec<u8>, VerifyingKey>,
+    keys: HashMap<Vec<u8>, Vec<u8>>,
 }
 
 impl TrustList {
-    /// Returns a given key by kid
-    pub fn get_key(&self, kid: &[u8]) -> Option<&VerifyingKey> {
+    /// Returns the public key with the specified key identifier or
+    /// [`None`] if there is no key with that key ID.
+    pub fn get_key(&self, kid: &[u8]) -> Option<&Vec<u8>> {
```

Comment:
Maybe we should return `Option<&[u8]>` at this point.

Reply:
Yes that's probably better, at the time I just search & replaced the old type.